### ISSUE TITLE
Error loading /etc/nomad.d/client.hcl: illegal char

### ISSUE THREAD

**Issue by heri16** [ansible-community/ansible-nomad#66]
I am getting an "illegal char error" with my client.hcl when using `nomad_chroot_env`.

This line seems to be incorrect:
https://github.com/brianshumate/ansible-nomad/blob/b37e501c5ff320e85fc10bc4d638285aec027867/templates/client.hcl.j2#L35

## What the template generated

```
client {
    enabled = true

    chroot_env = {
    "/etc/local.resolv.conf": "/etc/resolv.conf",
    "/run/systemd/resolve": "/run/systemd/resolve"
}
    }
```

## What the template should have generated

```
client {
    enabled = true

    chroot_env = {
    "/etc/local.resolv.conf" = "/etc/resolv.conf",
    "/run/systemd/resolve" = "/run/systemd/resolve"
}
    }
```

**Comment by heri16** [comment]
Looks like we have the same issue with `nomad_config_custom`:

https://github.com/brianshumate/ansible-nomad/blob/06a551249d15995c4c6ff69376063b407cd1c165/templates/custom.hcl.j2#L3

**Comment by tommyalatalo** [comment]
Just came here to mention the issue with nomad_chroot_env which I just stumbled upon. HCL and JSON are mixed in the client.hcl file.

nomad_config_custom however works for me, since that property creates a new file ("custom.hcl") and is not mixing hcl and json in the file contents. The custom.hcl file is wrong in the sense that it is actually json and not hcl, so yes, both of these should still be looked at.

**Comment by cricket007** [comment]
Closed with #69?

**Comment by lanefu** [comment]
assuming closed by #69
